src/components/AddComment.jsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from "react";
+import { postComment } from "../api";
+
+const AddComment = ({comments, review_id}) => {
+    const [commentAuthor, setCommentAuthor] = useState('');
+    const [commentBody, setCommentBody] = useState('');
+    const [commentVotes, setCommentVotes] = useState(0);
+    const [commentPosted, setCommentPosted] = useState(false)
+
+    return <section>
+        <form className="add-comment-form" onSubmit={(event) => {
+            event.preventDefault()
+
+            const commentToAdd = {
+                "author": commentAuthor,
+                "body": commentBody,
+                "votes": commentVotes
+            };
+            setCommentPosted(true)
```

Comment:
Should commentPosted be true at this point? We would want this to change only when the comment posting has gone through correctly. Where do we have access to this in our useEffect?